roles/jupyter/templates/jupyterhub_config.py.j2

```diff
@@ -4,11 +4,11 @@ import sys
 
 c.JupyterHub.admin_access = True
 c.JupyterHub.services = [
-    {
-        'name': 'experiment-launcher',
-        'admin': True,
-        'api_token': '{{ lookup('password', 'jupyterhub.launcher.token length=32 chars=ascii_letters,digits') }}',
-    },
+    # {
```

Comment:
this is now commented out. want to add a line explaining what it does if you uncomment it?